### PR TITLE
Feat/zIndex

### PR DESCRIPTION
**Pull request by xiaoyu-a**
## 📝 Description

Add a brief description.

## 💣 Is this a breaking change (Yes/No):

- [ ] Yes
- [x] No

## 🚧 How to migrate?

Don't need.

## 📝 Additional Information

lol

### CODE REVIEW

File: packages/input-tag/stories/input-tag.stories.tsx [hunk 1]
```diff
@@ -11,6 +11,24 @@ export default {
         type: "text",
       },
     },
+    borderColor: {
```
Comment (smallSohoSolo) [inline packages/input-tag/stories/input-tag.stories.tsx:14]: 没string？

File: packages/input/src/interface.tsx [hunk 2]
```diff
@@ -196,7 +196,7 @@ export interface StateValue {
   focus?: boolean
   variant?: string
   size?: InputProps["size"]
-  borderColor?: InputProps["borderColor"]
+  borderColor: InputBorderColor
```
Comment (smallSohoSolo) [inline packages/input/src/interface.tsx:199]: ？没了

File: packages/input/stories/input.stories.tsx [hunk 3]
```diff
@@ -2,6 +2,9 @@ import { Meta, Story } from "@storybook/react"
 import { Input, InputProps } from "../src"
 import { Space } from "@illa-design/space"
 import { PersonIcon } from "@illa-design/icon"
+import { useRef, useState } from "react"
+import { getColor } from "@illa-design/theme"
+import { colorPalette } from "@illa-design/theme/src"
```
Comment (smallSohoSolo) [inline packages/input/stories/input.stories.tsx:7]: 引用错了

File: packages/theme/src/color-palette.ts [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+) => {
+  const innerFormat = getFormat(format)
+
+  if (innerFormat === "hex") {
+    return color[innerFormat]()
+  }
+  return color[innerFormat]().round().string()
+}
+
+// 动态梯度算法
```
Comment (smallSohoSolo) [inline packages/theme/src/color-palette.ts:28]: 中文

**Review by Z-AOAO — APPROVED** [review]

File: packages/input-tag/stories/input-tag.stories.tsx (hunk 1 above)
Comment (xiaoyu-a) [reply packages/input-tag/stories/input-tag.stories.tsx:14]: 直接改成string吧

**Review by smallSohoSolo — APPROVED** [review]

**Pull request merged**